vispy/visuals/mesh.py

```diff
@@ -28,12 +29,10 @@
 varying vec4 v_light_color;
 varying vec4 v_base_color;
 
-
 void main() {
-
     v_ambientk = $ambientk;
     v_light_color = $light_color;
-    v_base_color = $base_color;
+    v_base_color = $color_transform($base_color);
```

Comment:
so here before we used to just use the variable `$base_color`, but I needed to add a color transform on top of that to use the colormap mapping